docs/building/README.md

```diff
@@ -13,6 +13,19 @@ Building is a necessary optimization when shipping apps to production. Generally
 
 Our recommended setups address the first four points but refrain from introducing non-standard features. We think it's best to avoid those, as they might lock your source code to a specific build setup which can be hard to get away from at a later time. Javascript is evolving fast, your code can quickly become out of sync with new developments.
 
+## Using `index.html` as an entry point
+Our configs for `rollup` & `webpack` are unique in that sense that they take a single html file as an entry point.
+Doing so allows you to work with the same entry point no matter if you use
+- webpack
+- rollup
+- owc-dev-server
+- polymer serve
+- ...
+
+This means you can easily compare how different setups effect your app size and loading time.
+So this truely means you can use `owc-dev-server` for your regular development then use `rollup` to prepare it to be put on a server.
+And if at some point you need to make an emergency fix that requires a dependecy that is not available as an es module just switch to `webpack ` for a few days. Make sure to request an es module version or make a temporary fork and switch back to `rollup` for optimal performance.
```

Comment:
```suggestion
And if at some point you need to make an emergency fix that requires a dependency that is not available as an es module, you can easily switch to `webpack` for a few days. (Make sure to request an es module version or make a temporary fork and switch back to `rollup` for optimal performance!)
```

Reply:
Also, rollup can handle commonjs right? Isnt this a bit of a weird example? As a developer im not very attracted to the idea of having to switch between build tools (even with open-wc recommendations/configs)